backend/src/models/service.ts

```diff
@@ -72,7 +72,6 @@ export interface Product {
 
 @Entity()
 @Index(['port', 'domain'], { unique: true })
-@Index(['domain'])
```

Comment:
Why remove this?

Reply:
it was giving a lot of "missing index". Researching I found that in this case I can create the index directly on the column definition. That seems to fix it.

Reply:
so the index is still there. It is just created differently.

Reply:
See line 85.